robolectric-resources/src/main/java/org/robolectric/manifest/AndroidManifest.java

```diff
@@ -152,6 +152,18 @@ void parseAndroidManifest() {
         }
 
         minSdkVersion = getTagAttributeIntValue(manifestDocument, "uses-sdk", "android:minSdkVersion");
+
+        String targetSdkText = getTagAttributeText(manifestDocument, "uses-sdk",
+            "android:targetSdkVersion");
+        if (targetSdkText != null) {
+          // Support Android O Preview. This can be removed once Android O is officially launched.
+          if (targetSdkText.length() == 1 && targetSdkText.equals("O")) {
+            targetSdkVersion = 26;
```

Comment:
Good catch, I forgot to remove the line below. Interestingly, the line below throws a NumberFormatException, which causes the second assignment not to happen. I'll remove the line below.